cisco_secure_email_threat_defense/README.md

```diff
@@ -1,43 +1,65 @@
-# Agent Check: cisco_secure_email_threat_defense
-
 ## Overview
 
-This check monitors [cisco_secure_email_threat_defense][1].
+[Cisco Secure Email Threat Defense][1] is an integrated cloud-native security solution for Microsoft 365 that focuses on simple deployment, easy attack remediation, and providing superior visibility into inbound, outbound, and internal user-to-user messages.
 
-## Setup
+This integration ingests the following logs:
+- Message: Message logs provide detailed information about email communications, including sender, recipient, timestamps, subject, and threat-related data for analysis and monitoring.
 
-### Installation
+The Cisco Secure Email Threat Defense integration provides out-of-the-box dashboards so you can gain insights into the Cisco Secure Email Threat Defense's message logs, enabling quick and necessary actions. Additionally, out-of-the-box detection rules are available to help you monitor and respond to potential security threats effectively.
 
-The Cisco Secure Email Threat Defense check is included in the [Datadog Agent][2] package.
-No additional installation is needed on your server.
+
+## Setup
 
 ### Configuration
 
-!!! Add list of steps to set up this integration !!!
+#### Get API Credentials for Cisco Secure Email Threat Defense 
+
+
+#### Steps to generate API key, Client ID and Client Password:
+1. Login to Cisco Secure Email Threat Defense UI.
+2. Navigate to `Administration` and select the `API Clients` tab.
+3. Click on `Add New Client` to generate the `Client ID` and `Client Password`.
+4. Enter the `Client Name` and an optional description, then click `Submit`. This will generate your Client ID and Password.
+    **Note**: Be sure to save the Client Password, as it will only be displayed once.
+5. Additionally, retrieve the `API key` from the API Key section, as the API key must be included in the header for the request.
```

Comment:
```suggestion
2. Navigate to **Administration** and select the **API Clients** tab.
3. Click on **Add New Client**.
4. Enter a **Client Name** and an optional description, then click **Submit**. This generates your Client ID and Password.
    **Note**: Be sure to save the Client Password, as it will only be displayed once.
5. Retrieve the API key from the **API Key** section.
```

Reply:
Updated as per suggestion